workers/ohsome_quality_analyst/cli.py

```diff
@@ -123,6 +124,7 @@ def create_indicator(
         )
         click.confirm("Do you want to continue?", abort=True)
     if infile is not None:
+        """When using an infile as input"""
         infile = pathlib.Path(infile)
         with open(infile, "r") as file:
```

Comment:
maybe also this part could be moved into a function called `load_infile` or similar. since it is used here and in the report as well. And in this function you can also do the check if the geometry is valid.

Reply:
I will add this function and replace the code